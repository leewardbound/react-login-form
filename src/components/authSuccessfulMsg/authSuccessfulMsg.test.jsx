import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import AuthSuccessfulMsg from '.';

const renderer = new ShallowRenderer();

describe('AuthSuccessfulMsg', function () {
  it('should render correctly', function () {
    const component = renderer.render(
      <AuthSuccessfulMsg />
    );

    expect(component).toMatchSnapshot();
  });
});
