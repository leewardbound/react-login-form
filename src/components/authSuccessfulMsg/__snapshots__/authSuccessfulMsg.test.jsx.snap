// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthSuccessfulMsg should render correctly 1`] = `
<Msg
  type="success"
>
  Login successful
</Msg>
`;
